### PR TITLE
Added column delivered to Order

### PR DESCRIPTION
**Pull request by VasiliBlr**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/controller/OrderController.java [hunk 1]
```diff
@@ -23,37 +22,48 @@
...
     }
 
     @PutMapping
-    public ResponseEntity<OrderDTO> updateOrder(@RequestBody OrderDTO orderDTO) {
+    public ResponseEntity<?> updateOrder(@RequestBody OrderDTO orderDTO) {
         return ResponseEntity.ok(orderService.saveOrder(orderDTO));
     }
+
+    @GetMapping("/{isDelivered}")
+    public ResponseEntity<?> getAllOrdersByDelivered(@PathVariable boolean isDelivered) {
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/OrderController.java:60]: getAllDeliveredOrders()

File: src/main/java/internship/mbicycle/storify/controller/OrderController.java [hunk 2]
```diff
@@ -23,37 +22,48 @@
...
+        return ResponseEntity.ok(orderService.getAllOrdersByDelivered(isDelivered));
+    }
+
+    @GetMapping("/{profileId}/{isDelivered}")
+    public ResponseEntity<?> getAllOrdersByProfileIdAndDelivered(@PathVariable Long profileId,
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/OrderController.java:65]: getAllDeliveredOrdersByProfileId()

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/controller/OrderController.java (hunk 1 above)
Comment (VasiliBlr) [reply src/main/java/internship/mbicycle/storify/controller/OrderController.java:60]: fixed

File: src/main/java/internship/mbicycle/storify/controller/OrderController.java (hunk 2 above)
Comment (VasiliBlr) [reply src/main/java/internship/mbicycle/storify/controller/OrderController.java:65]: fixed

**Pull request merged**